DEVELOPMENT.md

```diff
@@ -76,7 +76,7 @@ You can generate a distribution for your OS by running the following steps.
 From the project root, run the following commands
 
 - __Install the project dependencies__ by running `pnpm i`
-- __Build the UI and server__ by running `turbo build:local`
+- __Build the UI and server__ by running `turbo build:electron`
```

Comment:
I am unsure on what I missed, but the `build:electron` scripts seem like copies of `build:local`

I am unsure on how anything is different here?